tests/test_regression.py

```diff
@@ -485,6 +485,17 @@ def test_30_remove(self):
 
         expected = ['5.150.145.0/24', '109.161.64.0/20']
         self.assertEqual(expected, [n.prefix for n in tree])
+		
+	def test_31_covering_with_delete_error(self):
+        tree = radix.Radix()
+        tree.add('91.187.124.0/24')
+        tree.add('91.187.125.0/24')
+        tree.add('91.187.124.0/23')
+        tree.delete('91.187.124.0/23')
+        tree.add('91.187.124.0/23')
```

Comment:
Yep, I was expecting so. I don't have any idea about that why 91.187.124.0/23 is not in tree and I thought I was missing something.

Actually in version 0.9.2, this test gonna have this error in last .add
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/radix/radix.py", line 426, in add
    node = self._tree4.add(prefix)
  File "/usr/local/lib/python2.7/dist-packages/radix/radix.py", line 124, in add
    while node.bitlen < bitlen or node._prefix.addr is None:
AttributeError: 'NoneType' object has no attribute 'addr'

Thanks